gradle.properties

```diff
@@ -16,7 +16,7 @@ apacheCommonsCompressVersion=1.18
 apacheCuratorVersion=4.0.1
 apacheZookeeperVersion=3.5.5
 checkstyleToolVersion=8.23
-bookKeeperVersion=4.7.3
+bookKeeperVersion=4.9.2
```

Comment:
I'm not sure what you mean with this comment, @eolivelli .

Reply:
@fpj sorry for late reply
I was saying that the upgrade of BK should be performed in a separate patch.
In this patch I am doing two things: upgrade BK and move to the new API.
We could upgrade BK and then move to the new API in a follow up work

Reply:
PR #4420. That should go in before this one.

Reply:
@fpj I think the plan is to go with 4.9.2, as in #4519 
Honestly I think there is no value in staying in 4.8.x.
Hopefully we will release 4.11 in April and 4.8 will be far off the current active branches.
 
I can try to revert to 4.8 but the new API really started to be in good shape in 4.8 but it had some problem and since 4.9 it is the recommended one (in my personal opinion).

Reply:
@fpj We are upgrading to 4.9.2. There is nothing to revert here.